### DISCUSSION TITLE
Presentation.Slides[i] does not include Slide Notes?

### DISCUSSION THREAD

**Discussion by Joe-Blaze** [ShapeCrawler/ShapeCrawler#614] [Q&A]
Presentation2.Slides.Add(Presentation1.Slides[i]);
When copying a slide from a source presentation to a target presentation, the slide notes are not preserved.
Any suggestions would be appreciated.

**Comment by AdamShakhabov** [comment]
@Joe-Blaze, thank you for the report. Yes, It is a bug that must be fixed.

I've created a separated issue for this: https://github.com/ShapeCrawler/ShapeCrawler/issues/616

**Comment by Joe-Blaze** [comment]
Hi Adam:

Thank you for the speedy follow-up.
Hopefully this will not be a difficult bug to resolve.

Joe
________________________________

**Comment by jcoliz** [comment]
@Joe-Blaze, after #702 is merged, you will be able to at least add empty notes to the new slide, and copy over the note data, as a workaround.

**Comment by Joe-Blaze** [comment]
Thanks James - I'll check it out.
